air_pollution.ipynb

```diff
@@ -0,0 +1,1292 @@
+      "text/plain": [
+       "<Figure size 1440x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Correlation matrix\n",
+    "print('Pearson: ' + str(df_ap['pm2.5'].corr(df_ap['pm10'], method='pearson')))\n",
+    "print('Spearman: ' + str(df_ap['pm2.5'].corr(df_ap['pm10'], method='spearman')))\n",
+    "\n",
+    "#The highest correlation in the dataset is between PM10 and PM2.5 data \n",
+    "plt.figure(figsize=(20,10))\n",
+    "plt.scatter(df_ap[\"pm2.5\"], df_ap[\"pm10\"], alpha=.15, color='black')\n",
+    "plt.plot(np.unique(df_ap[\"pm2.5\"]), np.poly1d(np.polyfit(df_ap[\"pm2.5\"], df_ap[\"pm10\"], 1))\n",
+    "         (np.unique(df_ap[\"pm2.5\"])), color='red', linewidth=3, linestyle='dashed')\n",
+    "plt.xlabel('PM2.5 emission')\n",
+    "plt.ylabel('PM10 emission')\n",
+    "plt.title(\"Correlation between PM 10 and PM 2.5 values\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "12217fac",
+   "metadata": {},
+   "source": [
+    "<h4><font color=brown>Average PM2.5 per day</font></h4>"
```

Comment:
the table of content doesn't show anything for me

Reply:
![image](https://user-images.githubusercontent.com/118273022/203379993-ebc5ec06-38ec-419b-83fc-6b1f47f41b12.png)